configs/quality/technical_debt_audit_registry.yaml

```
@@ -66,3 +66,4 @@ audits:
     superseded_by: total-tech-debt-main-2026-07-19
     superseded_on: "2026-07-19"
     linked_issue: "#6355"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)